core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -189,9 +189,9 @@ public void testSchemaValidation() throws Exception {
     PythonEvaluator.Config config = new PythonEvaluator.Config(
       "def transform(x, emitter, context):\n" +
         " y = {}\n" +
-        "  y['intField'] *= 1024\n" +
-        "  y['longField'] *= 1024\n" +
-        "  emitter.emit(y)",
+        "y['intField'] *= 1024\n" +
```

Comment:
Seems like this was intentional by @gsps1: https://github.com/caskdata/hydrator-plugins/commit/7d4eb4d9cb445133b3277227a3d9abf896ce63b1#diff-9c235278978bbd8d97a04a785720692bR145